scripts/compile-pytorch-ipex.sh

```diff
@@ -144,13 +144,24 @@ if [ "$BUILD_PINNED" = true ]; then
   BUILD_IPEX=true
   if pip show intel_extension_for_pytorch &>/dev/null; then
     IPEX_CURRENT_COMMIT=`python -c "import torch;import intel_extension_for_pytorch as ipex;print(ipex.__version__)"`
-    IPEX_CURRENT_COMMIT=${IPEX_CURRENT_COMMIT#*"git"}
-    if [[ "$IPEX_PINNED_COMMIT" = "$IPEX_CURRENT_COMMIT"* ]]; then
-      echo "**** IPEX is already installed and its current commit is equal to the pinned commit: $IPEX_PINNED_COMMIT. ****"
-      BUILD_IPEX=false
+    if [ "$NO_OP_IPEX" = true ]; then
+      IPEX_CURRENT_COMMIT=${IPEX_CURRENT_COMMIT#*"+"}
+      if [[ "noop" = "$IPEX_CURRENT_COMMIT"* ]]; then
+        echo "**** No-op IPEX is already installed. ****"
+        BUILD_IPEX=false
+      else
+        echo "**** Current IPEX commit $IPEX_CURRENT_COMMIT ****"
+        echo "**** Expected no-op IPEX ****"
+      fi
     else
-      echo "**** Current IPEX commit $IPEX_CURRENT_COMMIT ****"
-      echo "**** Pinned IPEX commit $IPEX_PINNED_COMMIT ****"
+      IPEX_CURRENT_COMMIT=${IPEX_CURRENT_COMMIT#*"git"}
+      if [[ "$IPEX_PINNED_COMMIT" = "$IPEX_CURRENT_COMMIT"* ]]; then
```

Comment:
If you use double square brackets, then quotes are not needed:
```suggestion
      if [[ $IPEX_PINNED_COMMIT = $IPEX_CURRENT_COMMIT* ]]; then
```

Reply:
I see. Thank you!